src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import Header from './components/Header';
+import NavList from './components/NavList';
+import NavItem from './components/NavItem';
+import Backdrop from './components/Backdrop';
+import CloseButton from './components/CloseButton';
+
+const SIDEBAR_WIDTH = 256;
+
+const baseStyles = ({ theme, open }) => css`
+  label: sidebar;
+  display: flex;
```

Comment:
Do we need the flexbox here? If we're passing in block level elements anyway, those should stack up nicely with 100% width, right? And I don't see any use of `justify-content` or other flexbox features.

Reply:
Oh, that is right! I will remove it from the styles. Probably something that I was using on the beginning of the development